<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Particulas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Particulas</string>
          </property>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>51</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Id:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>60</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Origen en X:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>90</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Origen en Y:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>120</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Destino X:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>150</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Destino Y:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>180</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Velocidad:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>210</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Color:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>240</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Red:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>270</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Green:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>300</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Blue:</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="origenX_spinBox">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>60</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="origenY_spinBox">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>90</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="destinoX_spinBox">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>120</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="destinoY_spinBox">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>150</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="red_spinBox">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>240</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="green_spinBox">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>270</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="blue_spinBox">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>300</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
          <widget class="QPushButton" name="agregar_final_pushButton">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>350</y>
             <width>161</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>AGREGAR FINAL</string>
           </property>
          </widget>
          <widget class="QPushButton" name="agregar_inicio_pushButton">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>380</y>
             <width>161</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>AGREGAR INICIO</string>
           </property>
          </widget>
          <widget class="QPushButton" name="mostrar_pushButton">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>410</y>
             <width>161</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>MOSTRAR</string>
           </property>
          </widget>
          <widget class="QPlainTextEdit" name="salida">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>40</y>
             <width>231</width>
             <height>271</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="ordenar_Id_pushButton">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>350</y>
             <width>151</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>ORDENAR ID</string>
           </property>
          </widget>
          <widget class="QPushButton" name="ordenar_Distancia_pushButton">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>380</y>
             <width>151</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>ORDENAR DISTANCIA</string>
           </property>
          </widget>
          <widget class="QPushButton" name="ordenar_Velocidad_pushButton">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>410</y>
             <width>151</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>ORDENAR VELOCIDAD</string>
           </property>
          </widget>
          <widget class="QGraphicsView" name="graphicsView_Grafo">
           <property name="geometry">
            <rect>
             <x>450</x>
             <y>20</y>
             <width>531</width>
             <height>501</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="Profundidad_pushButton">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>440</y>
             <width>161</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>BUSQUEDA PROFUNDIDAD</string>
           </property>
          </widget>
          <widget class="QPushButton" name="Amplitud_pushButton">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>440</y>
             <width>151</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>BUSQUEDA AMPLITUD</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="spinBox">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>30</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QSpinBox" name="spinBox_2">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>180</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="3">
         <widget class="QTableWidget" name="tabla"/>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="buscar_lineEdit">
          <property name="placeholderText">
           <string>Id de la particula</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="mostrar_tabla_pushButton">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="buscar_pushButton">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Dibujar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="1">
         <widget class="QPushButton" name="limpiar">
          <property name="text">
           <string>Limpiar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="dibujar">
          <property name="text">
           <string>Dibujar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1028</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionPrim"/>
    <addaction name="actionKruskal"/>
    <addaction name="actionDijkstra"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPrim">
   <property name="text">
    <string>Algoritmo de Prim</string>
   </property>
  </action>
  <action name="actionKruskal">
   <property name="text">
    <string>Algoritmo de Kruskal</string>
   </property>
  </action>
  <action name="actionDijkstra">
   <property name="text">
    <string>Algoritmo de Dijkstra</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
